backend/hitas/migrations/0075_remove_realestate_street_address.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.18 on 2023-05-05 10:40
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("hitas", "0074_thirtyyearregulationresultsrow_letter_fetched"),
```

Comment:
Remember to fix this file migration number before merging this PR

For future reference, if you want separate PRs, but you already have a PR open that the second one is dependent (e.g. both create new migration files) you can have the second PR branch from the first PR's last commit, and then open the second PR to the first PR. That way when the first PR is merged, the second PR's target branch is automatically switched to `master`, and you dont need to remember to fix migration files 😉

Reply:
I opened separate PRs since either one can be merged first, and if one PR gets more comments than the other, it doesn't then block the merge of the other. Anyway, it's not ideal to work on multiple PRs that depend on each other at the same time, since if there are any conflicting changes to be made, it makes the rebase work harder. This can't be avoided always, but I'd rather have less of it than more.